.github/workflows/benchmark.yml

```diff
@@ -90,9 +92,61 @@ jobs:
       - name: Run
         run: |
           nix develop --no-warn-dirty --accept-flake-config --command bash ./scripts/ci-plutus-benchmark.sh 
-        if: success()
         env:
           GITHUB_TOKEN: ${{ github.token }}
           BENCHMARK_NAME: ${{ steps.extract-benchmark.outputs.benchmark }}
           PR_NUMBER: ${{ github.event.issue.number }}
           PR_BRANCH: ${{ steps.extract-branch.outputs.head_ref }}
+
+      - name: Publish Link To Action Run
```

Comment:
Can't we do this more simply with something like this: https://stackoverflow.com/questions/59073850/github-actions-get-url-of-test-build

Reply:
Also, I was thinking we should do this _before_ running the benchmark! The idea is that you get a link so you can see how it's progressing... which is not so useful once it's finished :)

Reply:
> Also, I was thinking we should do this before running the benchmark! The idea is that you get a link so you can see how it's progressing... which is not so useful once it's finished :)

Yes good catch, this should go before the `Run` step 😁 

> Can't we do this more simply with something like this: https://stackoverflow.com/questions/59073850/github-actions-get-url-of-test-build

Yes, something like that (e.g. https://github.com/input-output-hk/plutus/actions/runs/4102559646) is almost perfect, it was my starting point, but one still needs to click the `Benchmark` button to actually access the logs... I wanted to avoid that extra click.
I won't waste too much time on it though, testing this stuff locally is a nightmare